.copywrite.hcl

```diff
@@ -0,0 +1,20 @@
+schema_version = 1
+
+project {
+  license        = "MPL-2.0"
+  copyright_year = 2020
```

Comment:
Shouldn't this be `2023`?

Reply:
It defaults to the year the repo was created according to the [README](https://github.com/hashicorp/copywrite#config-structure) for the copywrite tool. This entry is optional though so we could exclude it if we wanted to.

> (OPTIONAL) Represents the year that the project initially began
  Default: <the year the repo was first created>
  copyright_year = 0